shm/src/game.cpp

```diff
@@ -17,6 +17,28 @@ Game::Game(size_t money, size_t days, size_t final_goal)
     print_cargo_command_ = std::make_unique<PrintCargo>();
 }
 
+void Game::startGame() {
+    while (days_ > time_->getElapsedTime()) {
+        if (checkWinCondition()) {
+            printWinScreen();
+            return;
+        } else if (checkLoseCondition()) {
+            printLooseScreen();
+            return;
```

Comment:
These two if's checking win / loose conditions should be at the end of the loop, because in case we make an action (for example travel) which will change the value of days_ such that the while loop will finish, we will not see the information if we lost or we won. This else is not necessary because of the return statement inside the if.

Reply:
Good point! Changed.